### DISCUSSION TITLE
Hero 12 = Max Lens Mod 2.0 + HDR = is possible

### DISCUSSION THREAD

**Discussion by LeckRab** [gopro/labs#749] [Q&A]
Hello friends!
Is it possible to make the following combination = Max Lens Mod 2.0 + HDR on the GoPro Hero12?
Works well?
How do I do?

**Comment by dnewman-gpsw** [comment]
It is currently not possible. My speculation: as there are two frames are read for the sensor to make an HDR composite, and the MLM requires the full sensor height to be read.  It is not possible to read the full sensor (8:7) at 60fps at the resolution needed to deliver 4K HDR with all the benefits of the MLM. The HDR modes are limit to 16x9 for the faster sensor modes, or one 4K 8:7 at lower frame rates 24/25/30.  If that mode was used for MLM the resolution would ~3K. There is the problem.

**Reply by LeckRab** [reply]
Great explanation! Thank you very much!

**Reply by kouyoumin** [reply]
The sensor can do 88fps @5599x4927x10bit according to [spec](https://www.sony-semicon.com/files/62/pdf/p-13_IMX677-AAPH5-J_Flyer.pdf).

**Reply by dnewman-gpsw** [reply]
Can it be DOL HDR into GP2? The whole system has be about to do, and fit within the thermal and power limits of the camera.
